drivers/input/button_upper.c

```diff
@@ -97,6 +97,7 @@ struct btn_open_s
    */
 
   FAR struct pollfd *bo_fds[CONFIG_INPUT_BUTTONS_NPOLLWAITERS];
+  bool bo_pending;
```

Comment:
```suggestion
  bool bo_pending;
  FAR struct pollfd *bo_fds[CONFIG_INPUT_BUTTONS_NPOLLWAITERS];
```
Just for consistency with other files

Reply:
Done.